src/Config/ObjectModel/RuntimeOptions.cs

```diff
@@ -3,4 +3,8 @@
 
 namespace Azure.DataApiBuilder.Config.ObjectModel;
 
-public record RuntimeOptions(RestRuntimeOptions Rest, GraphQLRuntimeOptions GraphQL, HostOptions Host);
+public record RuntimeOptions(RestRuntimeOptions Rest, GraphQLRuntimeOptions GraphQL, HostOptions Host, string? BaseRoute = null)
+{
+    public const string JSON_PROPERTY_NAME = "runtime";
+    public const string PROPERTY_NAME_BASE_ROUTE = "base-route";
```

Comment:
Does this affect OpenAPI document generation? Do we prepend the entity routes with base-route information?

Reply:
We'd need to append the base-route, if present, to the server property of the OpenApi doc:
https://github.com/Azure/data-api-builder/blob/bda0d6551f8f75fc7b98f888bdfce1f3dd317378/src/Core/Services/OpenAPI/OpenApiDocumentor.cs#L129-L132

Reply:
Thanks, added.